contracts/p1/BasketHandler.sol

```diff
@@ -255,7 +255,12 @@ contract BasketHandlerP1 is ComponentP1, IBasketHandler {
 
         for (uint256 i = 0; i < size; ++i) {
             CollateralStatus s = assetRegistry.toColl(basket.erc20s[i]).status();
-            if (s.worseThan(status_)) status_ = s;
+            if (s.worseThan(status_)) {
+                status_ = s;
+            }
+            if (status_ == CollateralStatus.DISABLED) {
```

Comment:
Not sure you saw my comment in the other PR but I think this change will end up being a net gas increase given how infrequently one of the backing will be DISABLED.

A change I like that might be the best of both worlds is putting this check inside the if statement above

Reply:
You are right, I was going to remove this, just missed it.